samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/LanguageGeneration.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.timex.resolution;
+
+import com.microsoft.recognizers.datatypes.timex.expression.TimexProperty;
+
+import java.time.LocalDateTime;
+
+/**
+ * This language generation capabilities are the logical opposite of what the recognizer does.
+ * As an experiment try feeding the result of language generation back into a recognizer.
+ * You should get back the same TIMEX expression in the result.
+ */
+public class LanguageGeneration{
```

Comment:
you might need to leave 1 space before the brace or Checkstyle will complain. But maybe we can do this when fixing the bot